package.json

```diff
@@ -17,7 +22,7 @@
 		"pretest": "npm run --silent lint && es-shim-api --bound",
 		"test": "npm run --silent tests-only",
 		"posttest": "npx aud",
-		"tests-only": "npm run --silent test:shimmed && npm run --silent test:module && npm run --silent tests:es5",
+		"tests-only": "npm run --silent test:shimmed && npm run --silent test:module && npm run --silent tests:es5 && npx tsc",
```

Comment:
does this not need `-p` to work?

Reply:
Interestingly enough, no. I assume this is one of the more recent changes in TypeScript.